tests/conftest.py

```diff
@@ -137,28 +137,29 @@ def glom(value, path):
     return g(value, re.sub(r"\[([0-9]*)\]", r".\1", path))
 
 
-@pytest.fixture
-def unique(request, freezer):
+def _get_prefix(request):
     test_class = request.cls
     if test_class:
-        prefix = "{}.{}".format(test_class.__name__, request.node.name)
+        main = "{}.{}".format(test_class.__name__, request.node.name)
     else:
-        prefix = request.node.name
+        base_name = request.node.__scenario_report__.scenario.name
```

Comment:
this assumes that we won't have non bdd tests. personally I am fine with that I only want to make sure that it is intentional.

Reply:
If you'e not in bdd you're in the first part of the condition no?